src/logic/operate.js

```diff
@@ -0,0 +1,30 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    try {
+      return one.div(two).toString();
+    } catch (err) {
+      return "Can't divide by 0.";
```

Comment:
- [ ] [REQUIRED] Good job so far! I can't finde `divide` operate on your project and `plus` operate display twice. Please fix it 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/220628235-4e76eb5a-8360-41ba-a73e-2c6b79e7dfcf.png)